frontend/public/components/dashboard/dashboards-page/cluster-dashboard/utilization-card.tsx

```diff
@@ -266,12 +281,26 @@ export const PrometheusMultilineUtilizationItem = withDashboardResources<
     namespace,
     isDisabled = false,
   }) => {
+    const { t } = useTranslation();
+
+    const UTILIZATION_QUERY_DURATION = {
```

Comment:
cant we use the `Duration` defined in the `duration-hook.ts` ? so we dont repeat ourselves